website/docs/guides/mattermost-integration_guide.md

```diff
@@ -0,0 +1,117 @@
+---
+id: mattermost-integration_guide
+title: Mattermost Integration
+sidebar_position: 7
+---
+
+## Overview
+
+Push enabled subscriptions provide us with the functionality to
+forward messages to mattermost channels via mattermost webhooks.
+
+#### Subscription Configuration
```

Comment:
please add a url to the main documentation and to the swagger API. 
Add a small description of Subscription configuration.